api/jest.utils.ts

```diff
@@ -85,10 +85,10 @@ export function createSuperRequest(config: {
   return (resource, options) => superRequest(resource, config, options);
 }
 
-export function setupServer(): void {
+export function setupServer(rateLimitTesting = false): void {
```

Comment:
I wanted to use env values for testing this, but was unsuccessful. Thus choose for the more safe option while testing. 

If you guys are okay with this please let me know! Otherwise I will try it again.

Rate limiting needs to be disabled for other tests other wise they will get a 429 code returned as well.